spec/tasks/migration_rake_spec.rb

```diff
@@ -1,484 +1,436 @@
+      expect(@file.institutional_visibility?).to be false
+      expect(@file.visibility_after_embargo).to eq "university_of_alberta"
+      expect(@file.embargo_release_date).to eq "Mon, 01 Jan 2114 07:00:00 +0000"
     end
   end
 
   describe 'migration:delete_by_uuids - delete' do
-    before do
+    before(:each) do
       Collection.delete_all
-      @community = Collection.new(title:'test community').tap do |c|
+      @community = Collection.new(title:'test community') do |c|
         c.apply_depositor_metadata('dittest@ualberta.ca')
         c.is_community = true
         c.is_official = true
         c.fedora3uuid = 'uuid:d04b3b74-211d-4939-9660-c390958fa2ee'
         c.save
         @id = c.id
       end
-      Rake::Task.define_task(:environment)
-      Rake::Task["migration:delete_by_uuid"].invoke('spec/fixtures/migration/uuids-to-delete')
     end
 
-    after do
-      Rake::Task["migration:delete_by_uuid"].reenable
-    end
-
-    subject { @id }
-
     it "community should have been deleted" do
-      expect {Collection.find(subject)}.to raise_error(Ldp::Gone)
+      run_rake_task('migration:delete_by_uuid', 'spec/fixtures/migration/uuids-to-delete')
+      expect {Collection.find(@id)}.to raise_error(Ldp::Gone)
     end
   end
+
   describe 'migration:update_ccid_visiblity' do
-    before do
+    before(:each) do
       GenericFile.delete_all
-      @generic_file = GenericFile.new(title:['test generic file']).tap do |c|
+      @generic_file = GenericFile.new(title:['test generic file']) do |c|
         c.apply_depositor_metadata('dittest@ualberta.ca')
         c.fedora3uuid = 'uuid:db49c90d-2788-4930-a71b-43fecc1b8bbd'
         c.save
       end
-      Rake::Task.define_task(:environment)
-      Rake::Task["migration:update_ccid_visiblity"].invoke('spec/fixtures/migration/uuids_ccid_visibility')
-      result = ActiveFedora::SolrService.instance.conn.get "select", params: {q:["fedora3uuid_tesim:uuid:db49c90d-2788-4930-a71b-43fecc1b8bbd"]}
-      doc = result["response"]["docs"].first
-      id = doc["id"]
-      @file = GenericFile.find(id)
-
-    end
-
-    after do
-      Rake::Task["migration:update_ccid_visiblity"].reenable
     end
 
-    subject { @file }
-
     it "visibility should have been updated" do
-      expect(subject.institutional_visibility?).to be true
-      expect(subject.read_groups).to include Hydranorth::AccessControls::InstitutionalVisibility::UNIVERSITY_OF_ALBERTA
-      expect(subject.read_groups).to include Hydra::AccessControls::AccessRight::PERMISSION_TEXT_VALUE_PUBLIC
+      run_rake_task('migration:update_ccid_visiblity', 'spec/fixtures/migration/uuids_ccid_visibility')
+      @generic_file.reload
+      expect(@generic_file.institutional_visibility?).to be true
+      expect(@generic_file.read_groups).to eq [Hydra::AccessControls::AccessRight::PERMISSION_TEXT_VALUE_PUBLIC, Hydranorth::AccessControls::InstitutionalVisibility::UNIVERSITY_OF_ALBERTA]
```

Comment:
I see we do alot of individual checking of arrays using the `to include <value>`. Why not just check if the array is exactly how it should be? 

Probably shouldn't be using eq matcher either. Should be using `match_array` matcher instead. Will fix this.